packages/hash/api/src/model/knowledge/user.model.ts

```diff
@@ -154,6 +154,10 @@ export default class extends EntityModel {
       isInstanceAdmin = false,
     } = params;
 
+    const hashInstanceModel = await HashInstanceModel.getHashInstanceModel(
```

Comment:
Why was this moved outside if we don't need it for adding an admin? Do we need to conduct any test?

Reply:
This wan't moved intentionally, reverted in https://github.com/hashintel/hash/pull/1489/commits/3b3308eea0904bfd51e50e7d79b864a44d213f3d